tests/csapi/apidoc_room_state_test.go

```diff
@@ -320,5 +321,18 @@ func TestRoomState(t *testing.T) {
 				},
 			})
 		})
+		t.Run("GET /rooms/:room_id/joined_members test to ensure user cannot request joined_members after leaving room", func(t *testing.T) {
+			t.Parallel()
+			roomID := authedClient.CreateRoom(t, map[string]interface{}{})
+			authedClient.LeaveRoom(t, roomID)
+			res := authedClient.DoFunc(t, "GET", []string{"_matrix", "client", "r0", "rooms", roomID, "joined_members"})
+			must.MatchResponse(t, res, match.HTTPResponse{
+				StatusCode: http.StatusForbidden,
+				JSON: []match.JSON{
+					match.JSONKeyEqual("errcode", "M_FORBIDDEN"),
+					match.JSONKeyEqual("error", "Getting joined members after leaving is forbidden."),
```

Comment:
The error message here isn't part of the spec---other homeservers could choose a different message (e.g. localised for the requester's language). I think we should remove this line.

```suggestion
```